src/ralph/ssl_certificates/management/commands/importssl_certificates.py

```diff
@@ -0,0 +1,68 @@
+import datetime
+import fnmatch
+import os
+from pathlib import Path
+
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
+from cryptography.x509.extensions import ExtensionNotFound
+from cryptography.x509.oid import ExtensionOID, NameOID
+from django.core.management.base import BaseCommand
+
+from ralph.assets.models.assets import AssetHolder
+from ralph.ssl_certificates.models import SSLCertificate
+
+
+class Command(BaseCommand):
+
+    help = 'Import data to application from dir'
+
+    def add_arguments(self, parser):
+        parser.add_argument('certs_dir', type=str)
+
+    def handle(self, *args, **options):
```

Comment:
could you split it into multiple smaller functions?

Reply:
As we talk, I'll try to do this in next pull request